courses/free-courses-en.md

```diff
@@ -602,6 +603,7 @@
 * [Bento JavaScript Learning Track](https://bento.io/topic/javascript) (Bento)
 * [Cycle.js Fundamentals](https://egghead.io/courses/cycle-js-fundamentals) - André Staltz
 * [ES6 and Typescript Tutorial](https://www.youtube.com/playlist?list=PLC3y8-rFHvwhI0V5mE9Vu6Nm-nap8EcjV) - Codevolution, Vishwas Gopinath (YouTube)
+* [Free Javascript Course](https://www2.ironhack.com/lp/free-courses/javascript) - IronHack
```

Comment:
Can't tell if this is free.
Don't invent titles! Use the title given by the resource.

Reply:
have you taken the course?